apps/rush-lib/src/cli/utilities/ChangeFiles.ts

```diff
@@ -54,6 +49,27 @@ export default class ChangeFiles {
     }
   }
 
+  public static getChangeComments(
+    newChangeFilePaths: string[],
+    changedPackages: string[]
+  ): Map<string, string[]> {
+    const changes: Map<string, string[]> = new Map<string, string[]>();
+
+    newChangeFilePaths.forEach((filePath) => {
+      console.log(`Found change file: ${filePath}`);
+      const changeRequest: IChangeInfo = JSON.parse(fsx.readFileSync(filePath, 'utf8'));
```

Comment:
>const changeRequest: IChangeInfo = JSON.parse(fsx.readFileSync(filePath, 'utf8')); [](start = 6, length = 82)

try/catch? how will you handle errors?

Reply:
If anything fails, let it throw and fail the action.  The filePath is outputted before this error. So it should be fairly easy to find out what is wrong with that file.

---
In reply to: [147840372](https://github.com/Microsoft/web-build-tools/pull/418#discussion_r147840372) [](ancestors = 147840372)